Tasks/CopyPublishBuildArtifacts/copypublishbuildartifacts.ts

```diff
@@ -85,130 +86,139 @@ var artifactType: string = tl.getInput('ArtifactType');
 var targetPath: string = tl.getInput('TargetPath');
 var findRoot: string = tl.getPathInput('CopyRoot');
 
-if (!artifactName) {
-    // nothing to do
-    tl.warning('Artifact name is not specified.');
-}
-else if (!artifactType) {
-    // nothing to do
-    tl.warning('Artifact type is not specified.');
-}
-else {
-    artifactType = artifactType.toLowerCase();
-    // back compat. remove after 82
-    if (artifactType === "localpath") {
-        artifactType = "filepath";
+let run = (): void => {
```

Comment:
Everything moved inside of a function because we either need to short-circuit or branch if/if-not a file share artifact type. Either way, the indention would change on (nearly) everything.